project/cmake/scripts/linux/install.cmake

```diff
@@ -19,6 +19,12 @@ configure_file(${CORE_SOURCE_DIR}/project/cmake/${APP_NAME_LC}-config.cmake.in
 
 configure_file(${CORE_SOURCE_DIR}/tools/Linux/kodi.sh.in
                ${CORE_BUILD_DIR}/scripts/${APP_NAME_LC} @ONLY)
+
+if (ENABLE_PULSEAUDIO AND PULSEAUDIO_FOUND)
+  file(READ "${CORE_SOURCE_DIR}/tools/Linux/kodi-standalone.sh.pulse" pulse_content)
+  set (XBMC_STANDALONE_SH_PULSE ${pulse_content})
+endif()
+
```

Comment:
Code looks good.
(usually we don't have spaces after if or set in cmake, but I won't insist)